lib/stitches/error.rb

```diff
@@ -1,9 +1,9 @@
 module Stitches
   class Error
     attr_reader :code, :message
-    def initialize(code:, message:)
-      @code    = code
-      @message = message
+    def initialize(options = {})
+      @code    = options[:code]
+      @message = options[:message]
```

Comment:
I think raising is a good idea for missing attributes. `fetch` would raise a `KeyError` which might not be the most descriptive.

What if an explicit `"Error must be initialized with a [code|message]"` error was raised?